### ISSUE TITLE
Mount options  to mount filesystem as network drive 

### ISSUE THREAD

**Issue by jslorrma** [Scille/winfspy#32]
## Context
I have written a custom HDFS (hadoop) filesystem using [fs_spec](https://github.com/fsspec/filesystem_spec) and used `winfspy` to mount the filesystem in Windows. I have adapted the [memfs-exmple](https://github.com/Scille/winfspy/blob/master/src/winfspy/memfs.py) from this repository accordingly. But I had to realize that Windows sends a lot of  `get_security_by_name` `open` ans `close` calls to the file system, which makes the mounted filesystem really slow. As suggested and discussed [here](https://github.com/winfsp/winfsp/issues/247), mounting a remote filesystem as a network drive (via `VolumePrefix`) should improve the mounting performance.

To my understanding in winfspy the `VolumePrefix` can be set via the `**volume_params` ->`prefix` argument of the FileSystem-constructor https://github.com/Scille/winfspy/blob/a786d8e28f06bc21a1accd4dd8ce16ac5b4364c0/src/winfspy/file_system.py#L105-L106 

`prefix` gets mapped to `FSP_FSCTL_VOLUME_PARAMS->Prefix[]` in: https://github.com/Scille/winfspy/blob/a786d8e28f06bc21a1accd4dd8ce16ac5b4364c0/src/_cffi_src/build_bindings.py#L173

## Problem
Mounting my filesystem with setting `prefix="\\server\hdfs"` or `prefix="/server/hdfs"` like:
```python
fs = FileSystem(
        str(mountpoint),
        operations,
        ...
        prefix="\\server\\hdfs"
)
```
does not have any effect and it is mounted as normal drive under the `mountpoint`.

## Question
Is this a bug? Or am I'm doing something wrong? How can I use `winfspy` to mount my filesystem as a network drive?

**Comment by vxgmichel** [comment]
Hi @jslorrma  and thanks for your report!

> Is this a bug?

It is! PR #33 should fix your issue, could you confirm that?

I tested using `python -m winfspy.memfs X: -p \Server\Share` and it seems to work fine.

**Comment by jslorrma** [comment]
Hey @vxgmichel,

that was quick 👍 Thanks!!

I successfully tested the proposed fix of #33 with the `memfs` example as well as with my `HDFS filesystem` implementation. And in fact, as presumed and discussed in https://github.com/winfsp/winfsp/issues/247, mounting the filesystem as a network drive significantly enhances the performance. 

Thanks once again.

**Comment by vxgmichel** [comment]
Alright, here's the release: [v0.8.3](https://github.com/Scille/winfspy/releases/tag/v0.8.3) :tada:
